requirements-dev.yml

```diff
@@ -6,6 +6,7 @@ dependencies:
   - python
   - ipython
   - jupyter
+  - jupyterlab
```

Comment:
I've included the packages required to run jupyter notebooks in the dev requirements, since those will be used for examples, as well as testing and documentation. But I'd like to keep other software that is not directly required for development out of the requirements files, since everyone has their own preferences.